creator-node/nginx_conf/nginx.conf

```diff
@@ -38,8 +38,8 @@ http {
             proxy_cache_use_stale error timeout updating http_500 http_502 http_503 http_504;
             proxy_cache_background_update on;
 
-            # Cache only 200 responses for 30m before considered stale
-            proxy_cache_valid 200 30m;
+            # Cache only 200 responses for some duration before considered stale
+            proxy_cache_valid 200 12h;
```

Comment:
what do the 2 diff changes here do? the `proxy_cache_path` and `proxy_cache_valid`? would you mind documenting, unclear from code

Reply:
updated comment

Reply:
🔥